admin_manual/configuration_server/occ_command.rst

```diff
@@ -1271,3 +1272,17 @@ option::
 You can perform this simulation manually with the ``--dry-run`` option::
  
  sudo -u www-data php occ upgrade --dry-run
+
+Two-factor authentication
+-------------------------
+If a two-factor provider app is enabled, it is enabled for all users by default
+(though the provider can decide whether or not the user has to pass the challenge).
+In the case of an user losing access to the second factor (e.g. lost phone with
+two-factor SMS verification), the admin can (temporarily) disable the two-factor
```

Comment:
remove the parenthesis in "(temporarily)".

In general from what I remember about technical writing, best is to avoid parenthesis at all. Try making these blocks into full sentences instead :smile: